src/test/java/com/google/step/servlets/HomePageServletTest.java

```diff
@@ -116,7 +116,7 @@ public void testDoGetAllDeals_success() throws Exception {
         String.format("[%s,%s,%s]", HOME_DEAL_A_JSON, HOME_DEAL_A_JSON, HOME_DEAL_A_JSON);
     String expected =
         String.format(
-            "{popularDeals:%s," + "usersIFollow:%s," + "restaurantsIFollow:%s," + "tagsIFollow:%s}",
+            "{trending:%s," + "users:%s," + "restaurants:%s," + "tags:%s}",
```

Comment:
```suggestion
            "{\"trending\":%s, \"users\":%s, \"restaurants\":%s, \"tags\":%s}",
```
Surround your json string keys with `"` and don't need the `+` between every attribute

Reply:
fix in #94